src/model-loader.js

```diff
@@ -2,7 +2,7 @@
 
 let modelLoader = {}
 
-modelLoader.loadModel = (action) => {
+modelLoader.load = (action, identity) => {
```

Comment:
Because of this
```
modelLoader.load = (action, identity) => {
  // TODO: LOAD THE SPECIFIED MODEL IF ALLOWED(ACCESSRULES.CHECKACCESS) AND EXPAND BASED ON SCHEMA.EXPANDABLES CONFIGURATION
  return null
}
```